cgo/kuzzle/c_to_go.go

```diff
@@ -141,27 +141,106 @@ func cToGoUserRigh(r *C.user_right) *types.UserRights {
 	return right
 }
 
-func cToGoSearchResult(s *C.search_result) *types.SearchResult {
-	opts := types.NewQueryOptions()
+func cToGoKuzzleRequest(r *C.kuzzle_request) *types.KuzzleRequest {
+	request := &types.KuzzleRequest{
+		RequestId:    C.GoString(r.request_id),
+		Controller:   C.GoString(r.controller),
+		Action:       C.GoString(r.action),
+		Index:        C.GoString(r.index),
+		Collection:   C.GoString(r.collection),
+		Id:           C.GoString(r.id),
+		From:         int(r.from),
+		Size:         int(r.size),
+		Scroll:       C.GoString(r.scroll),
+		ScrollId:     C.GoString(r.scroll_id),
+		Strategy:     C.GoString(r.strategy),
+		ExpiresIn:    int(r.expires_in),
+		Scope:        C.GoString(r.scope),
+		Users:        C.GoString(r.users),
+		Start:        int(r.start),
+		Stop:         int(r.stop),
+		End:          int(r.end),
+		Bit:          int(r.bit),
+		Member:       C.GoString(r.member),
+		Member1:      C.GoString(r.member1),
+		Member2:      C.GoString(r.member2),
+		Lon:          float64(r.lon),
+		Lat:          float64(r.lat),
+		Distance:     float64(r.distance),
+		Unit:         C.GoString(r.unit),
+		Cursor:       int(r.cursor),
+		Offset:       int(r.offset),
+		Field:        C.GoString(r.field),
+		Subcommand:   C.GoString(r.subcommand),
+		Pattern:      C.GoString(r.pattern),
+		Idx:          int(r.idx),
+		Min:          C.GoString(r.min),
+		Max:          C.GoString(r.max),
+		Limit:        C.GoString(r.limit),
+		Count:        int(r.count),
+		Match:        C.GoString(r.match),
+		Reset:        bool(r.reset),
+		IncludeTrash: bool(r.include_trash),
```

Comment:
Didn't we say we were depreciating this option?

Reply:
I was not aware it was, sorry.
By deprecating, do you mean it should be removed already from all Sdks or only not documented or marked as such?

Reply:
ok, I just checked and this option is used by `count`, `get` and `mget` only.
I agree it has no added-value. I'll remove it.

Reply:
We should mark it deprecated in the documentation v1 but maybe we could get ride of it directly in documenation v2 and especially in the SDKs